src/NuGet.Clients/Options/GeneralOptionControl.cs

```diff
@@ -98,5 +101,26 @@ private void packageRestoreConsentCheckBox_CheckedChanged(object sender, EventAr
                 packageRestoreAutomaticCheckBox.Checked = false;
             }
         }
+
+        private void localsCommandButton_OnClick(object sender, EventArgs e)
+        {
+            var arguments = new List<string>();
+            arguments.Add("all");
+            var settings = ServiceLocator.GetInstance<ISettings>();
+            var localsCommandRunner = new LocalsCommandRunner(arguments , settings, clear:true, list:false);
+            try
+            {
+                localsCommandRunner.ExecuteCommand();
+            }
+            catch(ArgumentException ex)
+            {
+                MessageBox.Show(ex.Message, Resources.PopupTitle_ClearNuGetCache);
```

Comment:
Spec : [link](https://github.com/NuGet/Home/wiki/Add-Clear-Cache-button-to-VS-Package-Manager-options)

Had a talk with hari about this. Will be making vhanges to the spec as well.